README.md

```diff
@@ -166,6 +171,39 @@ In fact, it is so common that we need to use a "last hour" parameter in jobs tha
 default offset for all the well known (aliased) schedules to +1 period. E.g for an `@hourly`
 schedule, the default offset is `PT1H`, and for a `@daily` schedule the offset is `P1D`
 
+#### `schedule[].secret` **[secret]**
+Secret is used to mount keys stored in [Kubernetes Secrets] into the container.
```

Comment:
Im thinking if maybe should stay on a higher level (meaning to abstract away the Kubernetes part)

maybe something like:
A Secret is a "container" for keys needed for the workflow container to run and have access to particular resources.

what do you think?